### ISSUE TITLE
Wait for tracks to be fully loaded

### ISSUE THREAD

**Issue by mikesafdieh** [igvteam/igv.js#738]
Is there any way to wait for all tracks to be loaded? I don't see an event for it on the wiki.

**Comment by jrobinso** [comment]
I would say no,  but can you define what you mean by wait and loaded?  What are you trying to do?

**Comment by mikesafdieh** [comment]
I'm trying to load multiple bigwig tracks and equalize the scale for each track by the track with the largest dataRange (after an autoscale). For example, in the image below, I want to load each track with a max range of 18:

<img width="1432" alt="screen shot igv" src="https://user-images.githubusercontent.com/8171274/46169049-12a52400-c268-11e8-8712-f04fa8bb754c.png">

Right now I have a button that I can click to do this (result below), but I want to be able to have this done when I first load the browser. 

<img width="1426" alt="screen shot igv scaled" src="https://user-images.githubusercontent.com/8171274/46169561-7d0a9400-c269-11e8-9e48-930a94233b37.png">

Additionally, I'd like to turn the "Equalize scale" button into a switch such that whenever selected, and there is a locus change, the browser will automatically autoscale and then equalize the scale (as shown above) when it reloads the tracks.

So what I need to know is when the dataRange autoscale values are ready for each track. Is there any kind of event for this?

**Comment by jrobinso** [comment]
No there's not,  that would not be possible.   They are independent and load as needed.  However you can group all these tracks to autoscale together.   See the "group autoscale"  example  (examples/groupAutoscale.html).    I think this does what you want.

**Comment by maya123z** [comment]
Is group autoscale no longer available? I'm not seeing the examples/groupAutoscale.html file referenced here.

**Comment by jrobinso** [comment]
Yes, it should work.  If you are having an issue post your track configuration.
